src/util/loanChannelUtils.js

```diff
@@ -170,7 +173,7 @@ export function watchChannelQuery(apollo, queryMap, channelUrl, loanQueryVars, n
 		experimentActive = checkCachedChannelExperiment(apollo);
 
 		if (experimentActive) {
-			observer = watchLoanChannel(apollo, queryMapFLSS, loanQueryVars);
+			observer = watchLoanChannel(apollo, { ...queryMapFLSS, ...selectedQuickFilters }, loanQueryVars);
```

Comment:
`{ ...queryMapFLSS, ...selectedQuickFilters }` can be stored in a const at the top of this method instead of needing to create this object twice in this method.